libsolidity/ast/Types.h

```diff
@@ -344,6 +344,91 @@ class IntegerConstantType: public Type
 };
 
 /**
+ * Fixed data type, compromised of M bits on left and N bits on right side of decimal.
+ */
+class RealType: public Type
+{
+public:
+	enum class Modifier
+	{
+		Unsigned, Signed
+	};
+	virtual Category category() const override { return Category::Real; }
+
+	explicit RealType(int M, int N, Modifier _modifier = Modifier::Unsigned);
```

Comment:
Please use something like `integerBits` and `fractionalBits` instead of `M` and `N`.